src/locales/en.json

```diff
@@ -0,0 +1,12 @@
+{
+  "Dev": {
```

Comment:
I have saved these texts. But they have changed and are not used yet:

- "Write to us"
- "Congratulations"
- "Details"
- "Description"